app/src/main/java/org/p2p/wallet/striga/signup/ui/StrigaSignUpSecondStepPresenter.kt

```diff
@@ -77,7 +82,18 @@ class StrigaSignUpSecondStepPresenter(
         val (isValid, states) = interactor.validateSecondStep(cachedSignupData)
 
         if (isValid) {
-            view?.navigateNext()
+            view?.setProgressIsVisible(true)
+            launch {
+                try {
+                    interactor.createUser()
+                    userInteractor.resendSmsForVerifyPhoneNumber().unwrap()
```

Comment:
I think create user already sends SMS for verification, no?

Reply:
nope, it only creates, but I guess it would be better to move it there